kubernetes/docs/ClusterInstallWithMicrok8s.md

```diff
@@ -50,6 +49,40 @@ microk8s kubectl label node <worker-node-name> worker=true
 ```
 microk8s kubectl uncordon <node-name>
 ```
+### Kubernetes Controller/Worker Cluster Internal IP Configuration Issue
+In certain scenarios when creating a cluster on the master of a federation, you may encounter the following problem:
+The cluster is configured with the external IP of the virtual machine (VM) rather than the VPN IP provided (typically in the format 10.86.x.10).
```

Comment:
Very specific. 

The problem is that when multiple network interfaces exist, for example while using a VPN, the wrong interface could be chosen, thus not allowing nodes to join the cluster. The problem is visible when the nodes become "Not Ready".